addons/binding/org.openhab.binding.cbus/src/main/java/org/openhab/binding/cbus/internal/cgate/CGateConnectException.java

```diff
@@ -0,0 +1,30 @@
+/**
+ *  CGateInterface - A library to allow interaction with Clipsal C-Gate.
```

Comment:
Thanks for your feedback @neohusky! If Scott isn't around, it would also be an option that you (or somebody else) pick up his work and drive it further. But I guess we first need clarification with Dave. Did you manage to reach him? If you didn't try yet, please feel free to do so!